tests/test_neut_const.py

```diff
@@ -33,6 +38,19 @@ def test_energy_conv(self):
         self.assertEqual(nc.eV_to_joule(1), 1.602e-19)
         self.assertEqual(nc.eV_to_wavelength_photon(1), 1.24e-6)
         self.assertEqual(nc.wavelength_to_meV_neutron(1), 81.81)
+        self.assertEqual(nc.joule_to_ev(1), (1/1.602e-19))
+
+
+class Z_dict_test_case(unittest.TestCase):
+    """ test for periodic table """
+
+    def test_table_len(self):
+        self.assertEqual((len(nc.Z_dict())), 118)
+                         
+    def test_last_value(self):

```

Comment:
this test checks the max value in the values, not the last entry, possibly change description